e2e/files.spec.ts

```diff
@@ -485,13 +506,20 @@ test.describe('Files', () => {
         await testAuth.step('go to trash can', async () => {
           await userPage.getByLabel('Trash Can').click()
 
-          const trashCanHeading = userPage.getByRole('heading', {
-            name: 'Trash Can',
-          })
-          await expect(trashCanHeading).toBeVisible()
+          // reload page if trash can deferred js does not load
```

Comment:
Safe to assume this probably intermittently fails because of resource constraints in CI, and not because of some other bug right?

Reply:
Yes I think so -- I saw this behavior (and similar timeouts waiting for network responses) on the linux runners, but haven't seen it on the macOS runners